simphony_ui/ui.py

```diff
@@ -110,8 +111,16 @@ class Application(HasStrictTraits):
     )
 
     @on_trait_change('calculation_error_event', dispatch='ui')
-    def show_error(self, msg):
-        error(None, 'Oups ! Something went bad:\n{}'.format(msg), 'Error')
+    def show_error(self, messages):
+        error_msg, tb_msg = messages
+        error(
+            None,
+            'Oups ! Something went bad:\n{}\n\nDetails:\n{}'.format(
```

Comment:
You can pass the detail as a special argument. The dialog will show a "more..." button
```
 71 def error(parent, message, title='Error', detail='', informative=''):              
```